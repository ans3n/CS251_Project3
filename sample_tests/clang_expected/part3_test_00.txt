insert 2kT0D Guldrise Thiralas
print
  0: [empty]
  1: [empty]
  2: 2kT0D -> Guldrise Thiralas
  3: [empty]
  4: [empty]
  5: [empty]
  6: [empty]
  7: [empty]
  8: [empty]
  9: [empty]
 10: [empty]
peek 2kT0D
Guldrise Thiralas
print
  0: [empty]
  1: [empty]
  2: 2kT0D -> Guldrise Thiralas
  3: [empty]
  4: [empty]
  5: [empty]
  6: [empty]
  7: [empty]
  8: [empty]
  9: [empty]
 10: [empty]
size
1
insert 5km1wQd Helseth Marvos
empty
false
insert 08NhG3y1 Talsi Ralvayn
size
3
empty
false
size
3
insert S9c0ISN Rathal Benethran
size
4
bucket_count
11
hash_code S9c0ISN
8
insert RhjfNWmS Raynilie Nithryon
insert ykUF6od Mandran Giral
peek 08NhG3y1
Talsi Ralvayn
bucket_count
11
empty
false
insert u1IZ5S Raynilie Marys
size
7
hash_code 5km1wQd
7
insert lrPeew Endar Dralayn
bucket_count
11
insert NYPtm Nilera Terandas
insert RZhaiz Baladas Hledri
bucket_count
11
peek u1IZ5S
Raynilie Marys
peek RZhaiz
Baladas Hledri
empty
false
hash_code u1IZ5S
2
bucket_count
11
insert AhNnOYaI Satyana Savani
print
  0: [empty]
  1: [empty]
  2: u1IZ5S -> Raynilie Marys
     └L: 2kT0D -> Guldrise Thiralas
  3: [empty]
  4: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: AhNnOYaI -> Satyana Savani
     └R: NYPtm -> Nilera Terandas
 10: 08NhG3y1 -> Talsi Ralvayn
insert s7ygSSJ Fevris Sadoro
peek u1IZ5S
Raynilie Marys
peek RZhaiz
Baladas Hledri
hash_code RhjfNWmS
5
insert Jjpux3KT Enar Relvani
insert 7QBzyC7H Drorayni Vendu
insert 7AOx Milyn Vari
print
  0: [empty]
  1: [empty]
  2: 7AOx -> Milyn Vari
     ├L: 2kT0D -> Guldrise Thiralas
     └R: Jjpux3KT -> Enar Relvani
      └R: u1IZ5S -> Raynilie Marys
  3: [empty]
  4: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: 7QBzyC7H -> Drorayni Vendu
     └R: s7ygSSJ -> Fevris Sadoro
      └L: AhNnOYaI -> Satyana Savani
       └R: NYPtm -> Nilera Terandas
 10: 08NhG3y1 -> Talsi Ralvayn
empty
false
peek RZhaiz
Baladas Hledri
insert uNZAES Falena Felas
insert alBK5xg Llathyno Othran
size
17
peek Jjpux3KT
Enar Relvani
print
  0: [empty]
  1: [empty]
  2: Jjpux3KT -> Enar Relvani
     ├L: 7AOx -> Milyn Vari
     │└L: 2kT0D -> Guldrise Thiralas
     └R: uNZAES -> Falena Felas
      └L: u1IZ5S -> Raynilie Marys
  3: alBK5xg -> Llathyno Othran
  4: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: 7QBzyC7H -> Drorayni Vendu
     └R: s7ygSSJ -> Fevris Sadoro
      └L: AhNnOYaI -> Satyana Savani
       └R: NYPtm -> Nilera Terandas
 10: 08NhG3y1 -> Talsi Ralvayn
insert WFJO7 Urandile Andrilo
print
  0: [empty]
  1: [empty]
  2: Jjpux3KT -> Enar Relvani
     ├L: 7AOx -> Milyn Vari
     │└L: 2kT0D -> Guldrise Thiralas
     └R: uNZAES -> Falena Felas
      └L: u1IZ5S -> Raynilie Marys
  3: alBK5xg -> Llathyno Othran
  4: RZhaiz -> Baladas Hledri
  5: WFJO7 -> Urandile Andrilo
     ├L: RhjfNWmS -> Raynilie Nithryon
     └R: lrPeew -> Endar Dralayn
  6: ykUF6od -> Mandran Giral
  7: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: 7QBzyC7H -> Drorayni Vendu
     └R: s7ygSSJ -> Fevris Sadoro
      └L: AhNnOYaI -> Satyana Savani
       └R: NYPtm -> Nilera Terandas
 10: 08NhG3y1 -> Talsi Ralvayn
peek lrPeew
Endar Dralayn
peek Jjpux3KT
Enar Relvani
insert pnleVCY Alms Velas
insert 9947X Manara Hardil
print
  0: [empty]
  1: [empty]
  2: Jjpux3KT -> Enar Relvani
     ├L: 7AOx -> Milyn Vari
     │└L: 2kT0D -> Guldrise Thiralas
     └R: uNZAES -> Falena Felas
      └L: u1IZ5S -> Raynilie Marys
  3: 9947X -> Manara Hardil
     └R: alBK5xg -> Llathyno Othran
      └R: pnleVCY -> Alms Velas
  4: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: WFJO7 -> Urandile Andrilo
      └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: 7QBzyC7H -> Drorayni Vendu
     └R: s7ygSSJ -> Fevris Sadoro
      └L: AhNnOYaI -> Satyana Savani
       └R: NYPtm -> Nilera Terandas
 10: 08NhG3y1 -> Talsi Ralvayn
bucket_count
11
insert eBIT Minasi Redas
insert Sew4 Guls Tobor
insert lDyAYuPO Talamu Ofemalen
peek lrPeew
Endar Dralayn
insert 13o5f Narile Salandas
print
  0: [empty]
  1: Sew4 -> Guls Tobor
  2: Jjpux3KT -> Enar Relvani
     ├L: 7AOx -> Milyn Vari
     │└L: 2kT0D -> Guldrise Thiralas
     └R: uNZAES -> Falena Felas
      └L: u1IZ5S -> Raynilie Marys
  3: 9947X -> Manara Hardil
     └R: alBK5xg -> Llathyno Othran
      └R: pnleVCY -> Alms Velas
  4: lDyAYuPO -> Talamu Ofemalen
     └L: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: WFJO7 -> Urandile Andrilo
      └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: 13o5f -> Narile Salandas
     └R: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: eBIT -> Minasi Redas
     ├L: 7QBzyC7H -> Drorayni Vendu
     │└R: NYPtm -> Nilera Terandas
     │ └L: AhNnOYaI -> Satyana Savani
     └R: s7ygSSJ -> Fevris Sadoro
 10: 08NhG3y1 -> Talsi Ralvayn
hash_code 5km1wQd
7
print
  0: [empty]
  1: Sew4 -> Guls Tobor
  2: Jjpux3KT -> Enar Relvani
     ├L: 7AOx -> Milyn Vari
     │└L: 2kT0D -> Guldrise Thiralas
     └R: uNZAES -> Falena Felas
      └L: u1IZ5S -> Raynilie Marys
  3: 9947X -> Manara Hardil
     └R: alBK5xg -> Llathyno Othran
      └R: pnleVCY -> Alms Velas
  4: lDyAYuPO -> Talamu Ofemalen
     └L: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: WFJO7 -> Urandile Andrilo
      └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: 13o5f -> Narile Salandas
     └R: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: eBIT -> Minasi Redas
     ├L: 7QBzyC7H -> Drorayni Vendu
     │└R: NYPtm -> Nilera Terandas
     │ └L: AhNnOYaI -> Satyana Savani
     └R: s7ygSSJ -> Fevris Sadoro
 10: 08NhG3y1 -> Talsi Ralvayn
hash_code NYPtm
9
print
  0: [empty]
  1: Sew4 -> Guls Tobor
  2: Jjpux3KT -> Enar Relvani
     ├L: 7AOx -> Milyn Vari
     │└L: 2kT0D -> Guldrise Thiralas
     └R: uNZAES -> Falena Felas
      └L: u1IZ5S -> Raynilie Marys
  3: 9947X -> Manara Hardil
     └R: alBK5xg -> Llathyno Othran
      └R: pnleVCY -> Alms Velas
  4: lDyAYuPO -> Talamu Ofemalen
     └L: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: WFJO7 -> Urandile Andrilo
      └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: 13o5f -> Narile Salandas
     └R: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: eBIT -> Minasi Redas
     ├L: 7QBzyC7H -> Drorayni Vendu
     │└R: NYPtm -> Nilera Terandas
     │ └L: AhNnOYaI -> Satyana Savani
     └R: s7ygSSJ -> Fevris Sadoro
 10: 08NhG3y1 -> Talsi Ralvayn
hash_code 13o5f
7
bucket_count
11
insert 4f2Q Lliros Molor
bucket_count
11
print
  0: [empty]
  1: Sew4 -> Guls Tobor
  2: Jjpux3KT -> Enar Relvani
     ├L: 7AOx -> Milyn Vari
     │└L: 2kT0D -> Guldrise Thiralas
     └R: uNZAES -> Falena Felas
      └L: u1IZ5S -> Raynilie Marys
  3: 9947X -> Manara Hardil
     └R: alBK5xg -> Llathyno Othran
      └R: pnleVCY -> Alms Velas
  4: lDyAYuPO -> Talamu Ofemalen
     └L: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: WFJO7 -> Urandile Andrilo
      └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: 13o5f -> Narile Salandas
     └R: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: eBIT -> Minasi Redas
     ├L: 7QBzyC7H -> Drorayni Vendu
     │└R: NYPtm -> Nilera Terandas
     │ └L: AhNnOYaI -> Satyana Savani
     └R: s7ygSSJ -> Fevris Sadoro
 10: 4f2Q -> Lliros Molor
     └L: 08NhG3y1 -> Talsi Ralvayn
peek pnleVCY
Alms Velas
insert 7Qnrvi Elmussa Samandas
bucket_count
11
insert fjgCe Faralenu Hlarar
insert cu56 Ferynu Relavel
peek uNZAES
Falena Felas
print
  0: [empty]
  1: cu56 -> Ferynu Relavel
     └L: Sew4 -> Guls Tobor
  2: uNZAES -> Falena Felas
     └L: u1IZ5S -> Raynilie Marys
      └L: fjgCe -> Faralenu Hlarar
       └L: Jjpux3KT -> Enar Relvani
        └L: 7Qnrvi -> Elmussa Samandas
         └L: 7AOx -> Milyn Vari
          └L: 2kT0D -> Guldrise Thiralas
  3: pnleVCY -> Alms Velas
     └L: alBK5xg -> Llathyno Othran
      └L: 9947X -> Manara Hardil
  4: lDyAYuPO -> Talamu Ofemalen
     └L: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: WFJO7 -> Urandile Andrilo
      └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: 13o5f -> Narile Salandas
     └R: 5km1wQd -> Helseth Marvos
  8: S9c0ISN -> Rathal Benethran
  9: eBIT -> Minasi Redas
     ├L: 7QBzyC7H -> Drorayni Vendu
     │└R: NYPtm -> Nilera Terandas
     │ └L: AhNnOYaI -> Satyana Savani
     └R: s7ygSSJ -> Fevris Sadoro
 10: 4f2Q -> Lliros Molor
     └L: 08NhG3y1 -> Talsi Ralvayn
peek 2kT0D
Guldrise Thiralas
insert l0zt Alms Llarys
print
  0: [empty]
  1: cu56 -> Ferynu Relavel
     └L: Sew4 -> Guls Tobor
  2: 2kT0D -> Guldrise Thiralas
     └R: u1IZ5S -> Raynilie Marys
      ├L: Jjpux3KT -> Enar Relvani
      │├L: 7AOx -> Milyn Vari
      ││└R: 7Qnrvi -> Elmussa Samandas
      │└R: fjgCe -> Faralenu Hlarar
      └R: uNZAES -> Falena Felas
  3: pnleVCY -> Alms Velas
     └L: alBK5xg -> Llathyno Othran
      └L: 9947X -> Manara Hardil
  4: lDyAYuPO -> Talamu Ofemalen
     └L: RZhaiz -> Baladas Hledri
  5: lrPeew -> Endar Dralayn
     └L: WFJO7 -> Urandile Andrilo
      └L: RhjfNWmS -> Raynilie Nithryon
  6: ykUF6od -> Mandran Giral
  7: l0zt -> Alms Llarys
     └L: 5km1wQd -> Helseth Marvos
      └L: 13o5f -> Narile Salandas
  8: S9c0ISN -> Rathal Benethran
  9: eBIT -> Minasi Redas
     ├L: 7QBzyC7H -> Drorayni Vendu
     │└R: NYPtm -> Nilera Terandas
     │ └L: AhNnOYaI -> Satyana Savani
     └R: s7ygSSJ -> Fevris Sadoro
 10: 4f2Q -> Lliros Molor
     └L: 08NhG3y1 -> Talsi Ralvayn
peek 7QBzyC7H
Drorayni Vendu
insert zlmDq Fanuse Salavel
quit
