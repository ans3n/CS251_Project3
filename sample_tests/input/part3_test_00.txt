string name
11
insert 2kT0D Guldrise Thiralas
print
peek 2kT0D
print
size
insert 5km1wQd Helseth Marvos
empty
insert 08NhG3y1 Talsi Ralvayn
size
empty
size
insert S9c0ISN Rathal Benethran
size
bucket_count
hash_code S9c0ISN
insert RhjfNWmS Raynilie Nithryon
insert ykUF6od Mandran Giral
peek 08NhG3y1
bucket_count
empty
insert u1IZ5S Raynilie Marys
size
hash_code 5km1wQd
insert lrPeew Endar Dralayn
bucket_count
insert NYPtm Nilera Terandas
insert RZhaiz Baladas Hledri
bucket_count
peek u1IZ5S
peek RZhaiz
empty
hash_code u1IZ5S
bucket_count
insert AhNnOYaI Satyana Savani
print
insert s7ygSSJ Fevris Sadoro
peek u1IZ5S
peek RZhaiz
hash_code RhjfNWmS
insert Jjpux3KT Enar Relvani
insert 7QBzyC7H Drorayni Vendu
insert 7AOx Milyn Vari
print
empty
peek RZhaiz
insert uNZAES Falena Felas
insert alBK5xg Llathyno Othran
size
peek Jjpux3KT
print
insert WFJO7 Urandile Andrilo
print
peek lrPeew
peek Jjpux3KT
insert pnleVCY Alms Velas
insert 9947X Manara Hardil
print
bucket_count
insert eBIT Minasi Redas
insert Sew4 Guls Tobor
insert lDyAYuPO Talamu Ofemalen
peek lrPeew
insert 13o5f Narile Salandas
print
hash_code 5km1wQd
print
hash_code NYPtm
print
hash_code 13o5f
bucket_count
insert 4f2Q Lliros Molor
bucket_count
print
peek pnleVCY
insert 7Qnrvi Elmussa Samandas
bucket_count
insert fjgCe Faralenu Hlarar
insert cu56 Ferynu Relavel
peek uNZAES
print
peek 2kT0D
insert l0zt Alms Llarys
print
peek 7QBzyC7H
insert zlmDq Fanuse Salavel
quit
