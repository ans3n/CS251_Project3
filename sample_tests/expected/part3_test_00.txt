insert 2kT0D Guldrise Thiralas
print
  0: [empty]
  1: [empty]
  2: [empty]
  3: 2kT0D -> Guldrise Thiralas
  4: [empty]
  5: [empty]
  6: [empty]
  7: [empty]
  8: [empty]
  9: [empty]
 10: [empty]
peek 2kT0D
Guldrise Thiralas
print
  0: [empty]
  1: [empty]
  2: [empty]
  3: 2kT0D -> Guldrise Thiralas
  4: [empty]
  5: [empty]
  6: [empty]
  7: [empty]
  8: [empty]
  9: [empty]
 10: [empty]
size
1
insert 5km1wQd Helseth Marvos
empty
false
insert 08NhG3y1 Talsi Ralvayn
size
3
empty
false
size
3
insert S9c0ISN Rathal Benethran
size
4
bucket_count
11
hash_code S9c0ISN
6
insert RhjfNWmS Raynilie Nithryon
insert ykUF6od Mandran Giral
peek 08NhG3y1
Talsi Ralvayn
bucket_count
11
empty
false
insert u1IZ5S Raynilie Marys
size
7
hash_code 5km1wQd
7
insert lrPeew Endar Dralayn
bucket_count
11
insert NYPtm Nilera Terandas
insert RZhaiz Baladas Hledri
bucket_count
11
peek u1IZ5S
Raynilie Marys
peek RZhaiz
Baladas Hledri
empty
false
hash_code u1IZ5S
7
bucket_count
11
insert AhNnOYaI Satyana Savani
print
  0: NYPtm -> Nilera Terandas
  1: RZhaiz -> Baladas Hledri
  2: [empty]
  3: 2kT0D -> Guldrise Thiralas
  4: [empty]
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: S9c0ISN -> Rathal Benethran
  7: u1IZ5S -> Raynilie Marys
     └L: 5km1wQd -> Helseth Marvos
  8: AhNnOYaI -> Satyana Savani
  9: RhjfNWmS -> Raynilie Nithryon
 10: [empty]
insert s7ygSSJ Fevris Sadoro
peek u1IZ5S
Raynilie Marys
peek RZhaiz
Baladas Hledri
hash_code RhjfNWmS
9
insert Jjpux3KT Enar Relvani
insert 7QBzyC7H Drorayni Vendu
insert 7AOx Milyn Vari
print
  0: 7QBzyC7H -> Drorayni Vendu
     └R: NYPtm -> Nilera Terandas
  1: RZhaiz -> Baladas Hledri
  2: [empty]
  3: Jjpux3KT -> Enar Relvani
     └L: 2kT0D -> Guldrise Thiralas
  4: [empty]
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: u1IZ5S -> Raynilie Marys
     └L: 5km1wQd -> Helseth Marvos
  8: AhNnOYaI -> Satyana Savani
  9: s7ygSSJ -> Fevris Sadoro
     └L: RhjfNWmS -> Raynilie Nithryon
 10: [empty]
empty
false
peek RZhaiz
Baladas Hledri
insert uNZAES Falena Felas
insert alBK5xg Llathyno Othran
size
17
peek Jjpux3KT
Enar Relvani
print
  0: 7QBzyC7H -> Drorayni Vendu
     └R: NYPtm -> Nilera Terandas
  1: RZhaiz -> Baladas Hledri
  2: [empty]
  3: Jjpux3KT -> Enar Relvani
     ├L: 2kT0D -> Guldrise Thiralas
     └R: alBK5xg -> Llathyno Othran
  4: [empty]
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: u1IZ5S -> Raynilie Marys
     └L: 5km1wQd -> Helseth Marvos
  8: uNZAES -> Falena Felas
     └L: AhNnOYaI -> Satyana Savani
  9: s7ygSSJ -> Fevris Sadoro
     └L: RhjfNWmS -> Raynilie Nithryon
 10: [empty]
insert WFJO7 Urandile Andrilo
print
  0: 7QBzyC7H -> Drorayni Vendu
     └R: NYPtm -> Nilera Terandas
  1: RZhaiz -> Baladas Hledri
  2: [empty]
  3: Jjpux3KT -> Enar Relvani
     ├L: 2kT0D -> Guldrise Thiralas
     └R: alBK5xg -> Llathyno Othran
  4: [empty]
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: u1IZ5S -> Raynilie Marys
     └L: 5km1wQd -> Helseth Marvos
  8: uNZAES -> Falena Felas
     └L: AhNnOYaI -> Satyana Savani
  9: WFJO7 -> Urandile Andrilo
     ├L: RhjfNWmS -> Raynilie Nithryon
     └R: s7ygSSJ -> Fevris Sadoro
 10: [empty]
peek lrPeew
Endar Dralayn
peek Jjpux3KT
Enar Relvani
insert pnleVCY Alms Velas
insert 9947X Manara Hardil
print
  0: 9947X -> Manara Hardil
     ├L: 7QBzyC7H -> Drorayni Vendu
     └R: pnleVCY -> Alms Velas
      └L: NYPtm -> Nilera Terandas
  1: RZhaiz -> Baladas Hledri
  2: [empty]
  3: Jjpux3KT -> Enar Relvani
     ├L: 2kT0D -> Guldrise Thiralas
     └R: alBK5xg -> Llathyno Othran
  4: [empty]
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: u1IZ5S -> Raynilie Marys
     └L: 5km1wQd -> Helseth Marvos
  8: uNZAES -> Falena Felas
     └L: AhNnOYaI -> Satyana Savani
  9: WFJO7 -> Urandile Andrilo
     ├L: RhjfNWmS -> Raynilie Nithryon
     └R: s7ygSSJ -> Fevris Sadoro
 10: [empty]
bucket_count
11
insert eBIT Minasi Redas
insert Sew4 Guls Tobor
insert lDyAYuPO Talamu Ofemalen
peek lrPeew
Endar Dralayn
insert 13o5f Narile Salandas
print
  0: 9947X -> Manara Hardil
     ├L: 7QBzyC7H -> Drorayni Vendu
     └R: pnleVCY -> Alms Velas
      └L: NYPtm -> Nilera Terandas
  1: RZhaiz -> Baladas Hledri
  2: eBIT -> Minasi Redas
  3: Jjpux3KT -> Enar Relvani
     ├L: 2kT0D -> Guldrise Thiralas
     └R: alBK5xg -> Llathyno Othran
  4: lDyAYuPO -> Talamu Ofemalen
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: Sew4 -> Guls Tobor
     ├L: 5km1wQd -> Helseth Marvos
     └R: u1IZ5S -> Raynilie Marys
  8: 13o5f -> Narile Salandas
     └R: AhNnOYaI -> Satyana Savani
      └R: uNZAES -> Falena Felas
  9: WFJO7 -> Urandile Andrilo
     ├L: RhjfNWmS -> Raynilie Nithryon
     └R: s7ygSSJ -> Fevris Sadoro
 10: [empty]
hash_code 5km1wQd
7
print
  0: 9947X -> Manara Hardil
     ├L: 7QBzyC7H -> Drorayni Vendu
     └R: pnleVCY -> Alms Velas
      └L: NYPtm -> Nilera Terandas
  1: RZhaiz -> Baladas Hledri
  2: eBIT -> Minasi Redas
  3: Jjpux3KT -> Enar Relvani
     ├L: 2kT0D -> Guldrise Thiralas
     └R: alBK5xg -> Llathyno Othran
  4: lDyAYuPO -> Talamu Ofemalen
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: Sew4 -> Guls Tobor
     ├L: 5km1wQd -> Helseth Marvos
     └R: u1IZ5S -> Raynilie Marys
  8: 13o5f -> Narile Salandas
     └R: AhNnOYaI -> Satyana Savani
      └R: uNZAES -> Falena Felas
  9: WFJO7 -> Urandile Andrilo
     ├L: RhjfNWmS -> Raynilie Nithryon
     └R: s7ygSSJ -> Fevris Sadoro
 10: [empty]
hash_code NYPtm
0
print
  0: 9947X -> Manara Hardil
     ├L: 7QBzyC7H -> Drorayni Vendu
     └R: pnleVCY -> Alms Velas
      └L: NYPtm -> Nilera Terandas
  1: RZhaiz -> Baladas Hledri
  2: eBIT -> Minasi Redas
  3: Jjpux3KT -> Enar Relvani
     ├L: 2kT0D -> Guldrise Thiralas
     └R: alBK5xg -> Llathyno Othran
  4: lDyAYuPO -> Talamu Ofemalen
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: Sew4 -> Guls Tobor
     ├L: 5km1wQd -> Helseth Marvos
     └R: u1IZ5S -> Raynilie Marys
  8: 13o5f -> Narile Salandas
     └R: AhNnOYaI -> Satyana Savani
      └R: uNZAES -> Falena Felas
  9: WFJO7 -> Urandile Andrilo
     ├L: RhjfNWmS -> Raynilie Nithryon
     └R: s7ygSSJ -> Fevris Sadoro
 10: [empty]
hash_code 13o5f
8
bucket_count
11
insert 4f2Q Lliros Molor
bucket_count
11
print
  0: 9947X -> Manara Hardil
     ├L: 7QBzyC7H -> Drorayni Vendu
     └R: pnleVCY -> Alms Velas
      └L: NYPtm -> Nilera Terandas
  1: RZhaiz -> Baladas Hledri
  2: eBIT -> Minasi Redas
  3: Jjpux3KT -> Enar Relvani
     ├L: 2kT0D -> Guldrise Thiralas
     └R: alBK5xg -> Llathyno Othran
  4: lDyAYuPO -> Talamu Ofemalen
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: Sew4 -> Guls Tobor
     ├L: 5km1wQd -> Helseth Marvos
     └R: u1IZ5S -> Raynilie Marys
  8: 13o5f -> Narile Salandas
     └R: AhNnOYaI -> Satyana Savani
      └R: uNZAES -> Falena Felas
  9: WFJO7 -> Urandile Andrilo
     ├L: RhjfNWmS -> Raynilie Nithryon
     └R: s7ygSSJ -> Fevris Sadoro
 10: 4f2Q -> Lliros Molor
peek pnleVCY
Alms Velas
insert 7Qnrvi Elmussa Samandas
bucket_count
11
insert fjgCe Faralenu Hlarar
insert cu56 Ferynu Relavel
peek uNZAES
Falena Felas
print
  0: pnleVCY -> Alms Velas
     └L: 9947X -> Manara Hardil
      ├L: 7QBzyC7H -> Drorayni Vendu
      └R: NYPtm -> Nilera Terandas
  1: fjgCe -> Faralenu Hlarar
     └L: RZhaiz -> Baladas Hledri
  2: eBIT -> Minasi Redas
  3: Jjpux3KT -> Enar Relvani
     ├L: 2kT0D -> Guldrise Thiralas
     └R: alBK5xg -> Llathyno Othran
  4: cu56 -> Ferynu Relavel
     └R: lDyAYuPO -> Talamu Ofemalen
  5: lrPeew -> Endar Dralayn
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: Sew4 -> Guls Tobor
     ├L: 5km1wQd -> Helseth Marvos
     └R: u1IZ5S -> Raynilie Marys
  8: uNZAES -> Falena Felas
     └L: AhNnOYaI -> Satyana Savani
      └L: 13o5f -> Narile Salandas
  9: WFJO7 -> Urandile Andrilo
     ├L: RhjfNWmS -> Raynilie Nithryon
     └R: s7ygSSJ -> Fevris Sadoro
 10: 7Qnrvi -> Elmussa Samandas
     └L: 4f2Q -> Lliros Molor
peek 2kT0D
Guldrise Thiralas
insert l0zt Alms Llarys
print
  0: pnleVCY -> Alms Velas
     └L: 9947X -> Manara Hardil
      ├L: 7QBzyC7H -> Drorayni Vendu
      └R: NYPtm -> Nilera Terandas
  1: fjgCe -> Faralenu Hlarar
     └L: RZhaiz -> Baladas Hledri
  2: eBIT -> Minasi Redas
  3: 2kT0D -> Guldrise Thiralas
     └R: Jjpux3KT -> Enar Relvani
      └R: alBK5xg -> Llathyno Othran
  4: cu56 -> Ferynu Relavel
     └R: lDyAYuPO -> Talamu Ofemalen
  5: l0zt -> Alms Llarys
     ├L: 08NhG3y1 -> Talsi Ralvayn
     └R: lrPeew -> Endar Dralayn
      └R: ykUF6od -> Mandran Giral
  6: 7AOx -> Milyn Vari
     └R: S9c0ISN -> Rathal Benethran
  7: Sew4 -> Guls Tobor
     ├L: 5km1wQd -> Helseth Marvos
     └R: u1IZ5S -> Raynilie Marys
  8: uNZAES -> Falena Felas
     └L: AhNnOYaI -> Satyana Savani
      └L: 13o5f -> Narile Salandas
  9: WFJO7 -> Urandile Andrilo
     ├L: RhjfNWmS -> Raynilie Nithryon
     └R: s7ygSSJ -> Fevris Sadoro
 10: 7Qnrvi -> Elmussa Samandas
     └L: 4f2Q -> Lliros Molor
peek 7QBzyC7H
Drorayni Vendu
insert zlmDq Fanuse Salavel
quit
